apps/admin/frontend/src/components/import_err_file_modal.tsx

```diff
@@ -26,6 +29,20 @@ const Content = styled.div`
   overflow: hidden;
 `;
 
+function errorCodeToMessage(
+  errorCode: ImportElectionResultsReportingError
+): string {
+  switch (errorCode.type) {
+    case 'parsing-failed':
+      return 'The results file could not be parsed.';
+    case 'conversion-failed':
+      return 'The contents of the file could not be converted.';
```

Comment:
I'm a little worried these error messages don't recommend any course of action for the user. I'm not totally sure how I would improve them though. What should they do if either of these errors occurs?

Reply:
Added some context in [a6cc1c0](https://github.com/votingworks/vxsuite/pull/5344/commits/a6cc1c0da44cf8db044956ea99b016705c37d632)

We can't make assumptions about where the ERR files are coming from so our help text unfortunately has to be a little vague. The text "Please re-export the results file from your system and import in VxAdmin again" would address file corruption or eg. pulling the USB drive out before writing finishes. It won't help with misconfiguration of or bugs in the exporting system, but without more information we can't really offer much advice.

Reply:
Also worth noting that the difference between "could not be parsed" and "could not be converted" have different technical implications, but the action for the user is the same. Parsing error == could not read file or file contents could not be parsed as JSON. Conversion error == file contents are valid JSON but did not match the ERR schema.

For conversion errors we could tell the user to look at the file and ensure it's a valid ERR file, but I'm not sure if we expect users to have fluency with the ERR format.

Reply:
That makes me think for parsing errors we could say something like: "File is unreadable. Try exporting it again." and for conversion errors we could say something like "File is not a valid Election Results Reporting CDF file. Please ensure you are using the correct file format."